src/utils/dependencies.utils.ts

```diff
@@ -27,6 +27,12 @@ export const loadInjectables = async (): Promise<void> => {
   for (const transformer of transformers) {
     await Injector.set('transformer', transformer.name, transformer.clazz, getClassDependencies(transformer.clazz));
   }
+
+  // Load middlewares.
+  const middlewares = Providers.getProvidersByType('middleware');
+  for (const middleware of middlewares) {
+    await Injector.set('middleware', middleware.name, middleware.clazz, getClassDependencies(middleware.clazz));
+  }
```

Comment:
Could use `Promise.all` like:
```ts
await Promise.all(middlewares.map(({ name, clazz }) => {
  return Injector.set('middleware', name, clazz, getClassDependencies(clazz))
}))
```

Reply:
mh right, this can be applicable to all the above loaders, right? I think it's a good idea

Reply:
Yes, makes sense.